templates/base.html

```diff
@@ -49,11 +49,15 @@
     {% endif %}
 
     {% if page.extra.thumbnail %}
-    <meta property="og:image" content="{{ get_url(path=page.path ~ page.extra.thumbnail) }}">
+      <meta property="og:image" content="{{ get_url(path=page.path ~ page.extra.thumbnail) }}">
     {% elif page.extra.static_thumbnail %}
-    <meta property="og:image" content="{{ get_url(path=page.extra.static_thumbnail) }}">
+      {%if page.extra.static_thumbnail is containing("http") %}
```

Comment:
Can you change this to use hasPrefix? If I had an thumbnail like `/static/http-example.jpg` it wouldn't work.

Reply:
Oh, good point! What about checking that it has "http://" or "https://"? I think that would be better than changing the name, agree?

Reply:
I think that could work :+1:

Reply:
Done

Reply:
Awesome thanks a bunch!